linera-views/src/views.rs

```diff
@@ -173,3 +175,16 @@ pub trait CryptoHashView<C>: HashableView<C> {
 /// A [`RootView`] that also supports crypto hash
 #[async_trait]
 pub trait CryptoHashRootView<C>: RootView<C> + CryptoHashView<C> {}
+
+/// A [`SharableView`] supports being shared (unsafely) by cloning it.
```

Comment:
Would `ClonableView` be a more obvious name here? I notice that all the explanations use the verb "clone".

Reply:
Done.